### ISSUE TITLE
Address header not folded with large number of recipients

### ISSUE THREAD

**Issue by Centridox** [PHPMailer/PHPMailer#2690]
Hi,

when adding multiple addresses on a To, CC or BCC, it seems that the result is not folded. All separate adresses are checked against the maximum length, but not the resulting output. This leads to an incorrect header being sent.

In the function addrAppend, there is just an implode, should the result of the implode (along with the other values) als be handled by the encodeHeader function?

Please advise

![image](https://user-images.githubusercontent.com/106731842/171580035-e41154d5-1f2d-49bb-99e1-600baf628f1e.png)

**Comment by Synchro** [comment]
Yes, you're right; It should be passed through encodeHeader too.

**Comment by Centridox** [comment]
The function encodeHeader also add encoding, how can i avoid that, and just trigger the linefolding?

**Comment by Synchro** [comment]
Some lines can't be folded safely without encoding, but it should only do that if it's necessary.
